### PR TITLE
17 use internal date formatter

### LINKED ISSUE #17: Use internal date formatter

### PR DESCRIPTION
**Pull request by batkedev**

### CODE REVIEW

File: src/components/HeatMap.jsx [hunk 1]
```diff
@@ -41,7 +40,7 @@ const HeatmapChart = ({ allWeather, currentDate }) => {
           if (
             tickDate.toDateString() === new Date(currentDate).toDateString()
           ) {
-            return format(tickDate, "HH");
+            return getHour(tickDate);
```
Comment (mannixcom) [inline src/components/HeatMap.jsx:44]: When I run the branch locally I get a getHour not defined error.

Comment (mannixcom) [reply src/components/HeatMap.jsx:44]: I have updated the function to tickDate.getHours() and it works for me locally.

Comment (batkedev) [reply src/components/HeatMap.jsx:44]: Ok, thanks!

An import from utils/date.js would fix it too.

The merge conflict caused the error, I suppose.

**Comment by batkedev** [comment]
You can merge it, if you like. :+1:

**Pull request merged**